src/Features/CSharp/Portable/xlf/CSharpFeaturesResources.zh-Hans.xlf

```diff
@@ -404,7 +404,7 @@
       </trans-unit>
       <trans-unit id="Make_0_return_Task_instead_of_void">
         <source>Make {0} return Task instead of void.</source>
-        <target state="translated">使 {0} 返回任务，而不是 void。</target>
+        <target state="translated">使 {0} 返回 Task，而不是 void。</target>
```

Comment:
`Task` is a type name and should not be translated.

Reply:
@huoyaoyuan The original resource should add single quotes so that the loc team doesn't localize it. I think ideally the original resource should be `'Make '{0}' return 'Task' instead of 'void'.`

Reply:
Here I relied on the `<source>` tag in xlf. If they mismatches, it should be a further tooling issues.

Reply:
@huoyaoyuan I meant to update this in resx and run UpdateXlf target to update all XLF files.